### ISSUE TITLE
Issue with  libgtk-3.so.0.2404.1 when using broadcom , segfault

### ISSUE THREAD

**Issue by kopax** [linuxmint/cinnamon#8929]
```
Cinnamon 3.8.8
Linux dka 4.19.0-6-amd64 #1 SMP Debian 4.19.67-2+deb10u1 (2019-09-20) x86_64 GNU/Linux
 ```

**Issue**

![image](https://user-images.githubusercontent.com/1866564/67801156-e1111480-fa88-11e9-8cb5-3ac0e66428cc.png)

This screen doesnt work, I can enable or disable, it always crash when I do `Network>Add new connection`, it is impossible to connect to my 4g network.

dmsg log this:

```
Oct 29 20:08:13 dka kernel: [  493.129468] cinnamon-settin[2268]: segfault at 20 ip 00007fe4c4562934 sp 00007ffd5a5e30a0 error 4 in libgtk-3.so.0.2404.1[7fe4c4295000+352000]
Oct 29 20:08:13 dka kernel: [  493.129475] Code: 84 00 00 00 00 00 49 8b 44 24 28 48 63 d5 48 89 14 24 48 8b 3c d0 e8 0b 98 d3 ff 49 8b 4c 24 28 48 8b 14 24 31 f6 66 0f ef c0 <48> 8b 78 20 0f 29 44 24 50 0f 29 44 24 60 0f 29 44 24 70 0f 29 84
```

Any idea what that mean?

**Comment by leigh123linux** [comment]
You need to report it to the Debian bug tracker not here.
They will need to fix it as 3.8.x cinnamon is EOL.

**Comment by arroadie** [comment]
Same issue running cinnamon 4.4 on arch
